packages/javascript-sdk/project.json

```diff
@@ -55,7 +55,47 @@
           "codeCoverage": true
         }
       }
+    },
+    "version": {
+      "executor": "@jscutlery/semver:version",
+      "options": {
+        "preset": "conventional",
+        "dryRun": true,
```

Comment:
Keeping as safety, we shouldn't run this anyways but i'd rather just have it as a backup